actions/workflows/create_vm_role.yaml

```diff
@@ -28,16 +39,16 @@ st2cd.create_vm_role:
             input:
                 key: ami_<% $.distro %>
             on-success:
-                - check_cname_if_it_already_exists
```

Comment:
I kept this as `check_cname` but was asked to change to `check_cname_if_it_already_exists` by @lakshmi-kannan.